testdata/kconfig_config.c

```diff
@@ -0,0 +1,73 @@
+#include "common.h"
+
+char __license[] __section("license") = "MIT";
+
+// Highly inspired from libbpf test:
```

Comment:
In that case the test would have to be GPL-2.0 as per the license header at the top of the file?